FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +20,50 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let _ = self else { return }
+			switch result {
+			case let .success((data, response)):
+				guard response.statusCode == RemoteFeedLoader.successStatusCode else {
+					completion(.failure(Error.invalidData))
+					return
+				}
+				guard let feed = try? JSONDecoder().decode(Feed.self, from: data) else {
+					completion(.failure(Error.invalidData))
+					return
+				}
+				completion(.success(feed.feedImages))
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+
+	private struct Feed: Decodable {
+		let items: [FeedImageAPI]
+
+		var feedImages: [FeedImage] {
+			items.compactMap { $0.feedImage }
+		}
+	}
+
+	private struct FeedImageAPI: Decodable {
+		let id: UUID
+		let description: String?
+		let location: String?
+		let urlString: String
```

Comment:
Following from my previous comment, `urlString` could be of type `URL`. That way `JSONDecoder` will take care of converting it to a valid `URL`, or throwing an error during `decode(_:from:)`
```suggestion
		let url: URL
```